index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
@marnen It’s really not a big issue—it’s just one word—and we are going to focus our time and effort elsewhere right now. Thanks for your input.